scripts/hid_configurator/configurator_core.py

```diff
@@ -92,30 +93,44 @@
     'desktop_mouse_nrf52832' : {
         'vid' : 0x1915,
         'pid' : 0x52DA,
-        'config' : PCA20044_CONFIG
+        'config' : PCA20044_CONFIG,
+        'stream' : False,
     },
     'desktop_mouse_nrf52810' : {
         'vid' : 0x1915,
         'pid' : 0x52DB,
-        'config' : PCA20045_CONFIG
+        'config' : PCA20045_CONFIG,
+        'stream' : False,
     },
     'gaming_mouse' : {
         'vid' : 0x1915,
         'pid' : 0x52DE,
-        'config' : PCA20041_CONFIG
+        'config' : PCA20041_CONFIG,
+        'stream' : True,
     },
     'keyboard' : {
         'vid' : 0x1915,
         'pid' : 0x52DD,
-        'config' : None
+        'config' : None,
+        'stream' : False,
     },
     'dongle' : {
         'vid' : 0x1915,
         'pid' : 0x52DC,
-        'config' : None
+        'config' : None,
+        'stream' : False,
     }
 }
 
+interrupted = False
+
+
+def signal_handler(signal, frame):
```

Comment:
move closer to while loop and avoid using globals if possible

Reply:
done